prompts/deepset-conversational-agent.md

```diff
@@ -0,0 +1,34 @@
+This prompt is intended to be used with a Haystack `Agent`, specificallly, a [`ConversationalAgent`](https://docs.haystack.deepset.ai/docs/agent#conversational-agent).
+
+To learn how to use this prompt and the `ConversationalAgent`, check out our [Building a Conversational Chat App Tutorial](https://haystack.deepset.ai/tutorials/24_building_chat_app).
+
+By default, the `ConversationalAgent` in Haystack would use this prompt without having to provide it using `prompt_template`. However, below is an example of how you might manually create a `PromptTemplate` with this prompt, and how you might use it with the ConversationalAgent.
```

Comment:
So which prompt does conversational agent use by default? Is it `deepset-conversational-agent` or `deepset-conversational-agent-without-tools`?

Reply:
Good question. Now that I think about it again, it's not correct to state that ConversationalAgent uses any of these prompt. It uses the **copy** of these prompts in haystack repo

Reply:
I think we can still say "use this prompt", because if we switch to "copy of this prompt", even though technically it's correct, it will complicate the language for the whole prompthub and we'll need to keep it in mind every time.